src/Approvals/SFA.DAS.Approvals.Api/Controllers/CohortController.cs

```diff
@@ -150,6 +151,30 @@ public async Task<IActionResult> GetAddDraftApprenticeshipCourse([FromQuery] lon
             }
         }
 
+        [HttpGet]
+        [Route("employer/{accountId}/unapproved/add/confirm-employer")]
+        [Route("provider/{providerId}/unapproved/add/confirm-employer")]
+        public async Task<IActionResult> GetHasDeclaredStandards([FromQuery] long? providerId)
```

Comment:
I think the naming of this method and all of the classes (response, query, query result, query handler) is confusing. The endpoint is here to support the UI's "confirm-employer" endpoint, so I think "ConfirmEmployer" should feature in the name of everything.